Gruntfile.js

```diff
@@ -96,6 +96,33 @@ module.exports = function(grunt) {
         src: ['<%= concat.dist.src %>', 'plugin/*.js']
       },
     },
+    jasmine: {
+      core: {
+        src: '<%= concat.dist.src %>',
+        options: {
+          specs: 'spec/*Spec.js',
```

Comment:
Sounds good. Will also try to get coveralls working on travis and rebase to a single commit when I'm done.